### PR TITLE
🐛  Fixed unable to delete member

### PR DESCRIPTION
**Pull request by rshbhgrg**
closes https://github.com/TryGhost/Team/issues/557

After updating to v4 on SQLite from v3, we were unable to delete members due to a foreign key mismatch error. This is because the migrations which recreates the tables for `members_stripe_customers_subscriptions` and `members_stripe_customers` doesn't add back the unique constraint which is needed by FKs for reference.

The migration creates the missing UNIQUE constraints on the tables.

### CODE REVIEW

File: core/server/data/migrations/versions/4.1/01-add-unique-constraint-for-member-stripe-tables.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const logging = require('../../../../../shared/logging');
+const {createIrreversibleMigration} = require('../../utils');
+const {addUnique} = require('../../../schema/commands');
+
+module.exports = createIrreversibleMigration(async (knex) => {
```
Comment (allouis) [inline core/server/data/migrations/versions/4.1/01-add-unique-constraint-for-member-stripe-tables.js:5]: We don't usually create irreversible migrations outside of a major, I think this should just have a `noop` for the down migration :thinking:

**Review by allouis — COMMENTED** [review]
Smol comment

File: core/server/data/migrations/versions/4.1/01-add-unique-constraint-for-member-stripe-tables.js (hunk 1 above)
Comment (rishabhgrg) [reply core/server/data/migrations/versions/4.1/01-add-unique-constraint-for-member-stripe-tables.js]: @allouis Yeah good point, have changed it to `transactionalMigration` now with noop for down migration

**Review by daniellockyer — APPROVED** [review]

**Review by allouis — APPROVED** [review]

**Review by tpatel — APPROVED** [review]
looks great 👍

**Pull request merged**